janusgrap cql copy resource failed for file not found error when compile in cgwin.

it has occured error(

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:2.6:copy-resources (filter-byteorderedpartitioner) on project janusgraph-cql: Error loading  property file 'src/test/resources/properties/byteorderedpartitioner.properties': src\test\resources\properties\byteorderedpartitioner.properties -> [Help 1]
--

) when i compile the whole project in cgwin.

Further,  it is show 'src/test/resources/properties/byteorderedpartitioner.properties' file not found when openning the debug mode.

when i use absolute path like this:

janusgraph-cql/pom.xml
--
-                src/test/resources    
+                ${basedir}/src/test/resources                                                                                                                                                                                                                                                                                                                                                              cassandra/                                                                                                                                                                               keystores/
--

it worked. 

Is this relative source path not in current build path(janusgraph-cql)?

--- a/pom.xml
+++ b/pom.xml
@@ -418,7 +418,7 @@
                 </plugin>
                 <plugin>
                     <artifactId>maven-resources-plugin</artifactId>
-                    <version>2.6</version>
+                    <version>3.1.0</version>
                 </plugin>
                 <plugin>
                     <artifactId>maven-release-plugin</artifactId>

